### PR TITLE
fix: login with handle

### PR DESCRIPTION
**Pull request by cesarenaldi**

### CODE REVIEW

File: examples/web-wagmi/src/authentication/LoginSpecificProfile.tsx [hunk 1]
```diff
@@ -18,95 +10,31 @@ function UnauthenticatedContent() {
...
-        })}
-      </select>
-
-      <button onClick={onLoginClick}>
-        <strong>Log in</strong>
-      </button>
-    </>
-  );
-}
-
```
Comment (cesarenaldi) [inline examples/web-wagmi/src/authentication/LoginSpecificProfile.tsx:104]: I simplified this example... when already logged-in, logging-in with another handle on the same wallet is not a use case. One should use `useActiveProfileSwitch` for that purpose.

File: packages/react/src/transactions/infrastructure/TransactionObserver.ts [hunk 2]
```diff
@@ -33,7 +33,7 @@ const ONE_SECOND = 1000; // ms
  * @internal
  */
 export type TransactionObserverTimings = {
-  pollingInterval: number; // ms // TODO delete this
+  pollingInterval: number; // ms // TODO move this definition closer to EnvironmentConfig, not used by TransactionObserver
```
Comment (cesarenaldi) [inline packages/react/src/transactions/infrastructure/TransactionObserver.ts:36]: I was confused to had to investigate. Left my findings.

File: packages/react/src/wallet/useWalletLogin.ts [hunk 3]
```diff
@@ -31,7 +64,7 @@ export function useWalletLogin(): WalletLoginOperation {
       if ('address' in walletLoginArgs) {
         return loginWallet({
           address: walletLoginArgs.address,
-          handle,
+          handle: walletLoginArgs.handle,
```
Comment (cesarenaldi) [inline packages/react/src/wallet/useWalletLogin.ts:67]: The fix.

Comment (reecejohnson) [reply packages/react/src/wallet/useWalletLogin.ts:67]: Ah, nice catch. Thanks

**Review by reecejohnson — APPROVED** [review]

**Pull request merged**